fwlitetest.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# Takes no params, must be called from test dir
+
+bd=`pwd | sed 's/-test.*//'`
+pr=`pwd | sed 's/.*PR//'`
+rA=testPR$pr; rB=orig;
+source ~/tools/validateJR.sh ${bd}-${rA} ${bd}-${rB} ${rA}VS${rB} ~/tools/matrix_70X.txt |& tee ${rA}VS${rB}.log 
```

Comment:
I don't find this one particularly useful, esp considering that rA and rB are often different from what you derive in the first two lines.
I wouldn't object much to merge it, just don't find it useful.